test/e2e/kubectl/kubectl.go

```diff
@@ -489,14 +486,6 @@ var _ = SIGDescribe("Kubectl client", func() {
 				if output != expectedExecOutput {
 					framework.Failf("Unexpected kubectl exec output. Wanted %q, got  %q", expectedExecOutput, output)
 				}
-
-				// Verify the proxy server logs saw the connection
-				expectedProxyLog := fmt.Sprintf("Accepting CONNECT to %s", strings.TrimSuffix(strings.TrimPrefix(framework.TestContext.Host, "https://"), "/api"))
-
-				proxyLog := proxyLogs.String()
-				if !strings.Contains(proxyLog, expectedProxyLog) {
-					framework.Failf("Missing expected log result on proxy server for %s. Expected: %q, got %q", proxyVar, expectedProxyLog, proxyLog)
-				}
```

Comment:
Can we retain these logs in `NewHTTPProxyHandler`?  I would prefer to limit changes to these tests.

Reply:
Done-ish using the hook.